src/EA.Weee.Web/Areas/Admin/Views/ManageEvidenceNotes/ViewTransferNote.cshtml

```diff
@@ -0,0 +1,24 @@
+﻿@using EA.Weee.Web.RazorHelpers
+@model EA.Weee.Web.ViewModels.Shared.ViewTransferNoteViewModel
+@{
+    ViewBag.Title = Model.TabName + " - GOV.UK";
```

Comment:
I dont think we need this view. Should be able to use the existing ViewEvidenceNoteTransfer and its view, if there is a reason you dont think we can reuse it let me know.